calicoctl/commands/get.go

```diff
@@ -61,7 +62,8 @@ Options:
   -a --all-namespaces          If present, list the requested object(s) across all namespaces.
   --export                     If present, returns the requested object(s) stripped of
                                cluster-specific information. This flag will be ignored
-			       if <NAME> is not specified.
```

Comment:
Indentation looks wrong here too? Although seems it was wrong in the other direction to begin with :D

Reply:
Nice catch!